Update file1.adoc (#13)
alxlenc/pygh-tests: content/file1.adoc

## This is a Title
I create a file.

> The scope of the OpenShift Container Platform installation program is designed for simplicity and ensured success.

Reviewer: How about...
```suggestion
The designers aim to ensure simplicity and success in the scope of the OpenShift Container Platform installation program.
```
